.github/workflows/api-docs.yml

```diff
@@ -0,0 +1,34 @@
+name: wiki-docs-generator
+
+on:
+  release:
+    types: [published, edited]
+
+jobs:
+  docs:
+    env:
+      RELEASE_TAG: ${{ github.event.release.tag_name }}
+      SPEC_FOLDER: api-specs/${{ github.event.release.tag_name }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Gen docs
+        run: make
+      - name: Checkout wiki code
+        uses: actions/checkout@v2
+        with:
+          repository: ${{github.repository}}.wiki
+      - name: add docs to api-spec.md
+        run: |
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
+          cd tapir.wiki
+          [ -d $SPEC_FOLDER ] || mkdir -p $SPEC_FOLDER
+          yes | cp ../api-docs/api-spec* $SPEC_FOLDER
+          cd $SPEC_FOLDER
+          echo "\n\n## $RELEASE_TAG \n" >> ../../api-spec.md
+          for n in api-spec-stroeer*; do filepath=${n/.md/}; label=${filepath/api-spec-/}; echo "• [$label]($SPEC_FOLDER/$filepath)" >> ../../api-spec.md; done
+          cd ../..
```

Comment:
I am sure there is much easier and shorter code for this part. so feedback is very welcome.